test/test.js

```diff
@@ -141,6 +142,39 @@ test('v-stream directive (basic)', done => {
   })
 })
 
+test('v-stream directive (with .native modify)', done => {
+  const vm = new Vue({
+    template: `
+      <div>
+        <span class="count">{{ count }}</span>
+        <my-button v-stream:click.native="clickNative$">+</my-button>
+        <my-button v-stream:click="click$">-</my-button>
+      </div>
+    `,
+    components: {
+      myButton: {
+        template: '<button>MyButton</button>'
+      }
+    },
+    domStreams: ['clickNative$', 'click$'],
+    subscriptions () {
+      return {
+        count: this.click$.map(() => -1)
+          .merge(this.clickNative$.map(() => 1))
```

Comment:
This map operator function should asserting the passing argument is indeed a native event